### PR TITLE
feat: expose `DataFrame.bqclient` to assist in integrations

### PR DESCRIPTION
**Pull request by tswast**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-bigquery-dataframes/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: notebooks/dataframes/dataframe.ipynb [hunk 1]
```diff
@@ -1,5 +1,27 @@
 {
  "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
```
Comment (junyazhang) [inline notebooks/dataframes/dataframe.ipynb:5]: Just curious, how much do we care about these `execution_count` in notebooks? We are having two execution_count = 1 in this file.

File: notebooks/dataframes/integrations.ipynb [hunk 2]
```diff
@@ -0,0 +1,445 @@
...
+    "    "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
```
Comment (junyazhang) [inline notebooks/dataframes/integrations.ipynb:422]: Is this empty cell (line 416-422) intended here?

**Review by junyazhang — APPROVED** [review]
Nice tutorial!

File: notebooks/dataframes/dataframe.ipynb (hunk 1 above)
Comment (tswast) [reply notebooks/dataframes/dataframe.ipynb:5]: I don't think we really care. I recommend using the ReviewNB link above to review as a rendered version: https://github.com/googleapis/python-bigquery-dataframes/pull/519#issuecomment-2021335773

File: notebooks/dataframes/integrations.ipynb (hunk 2 above)
Comment (tswast) [reply notebooks/dataframes/integrations.ipynb:612]: Good catch. I've removed the empty cells and made an update to the "Returning a BigQuery DataFrames (bigframes) DataFrame" sample.

**Pull request merged**